common/src/autogluon/common/utils/s3_utils.py

```diff
@@ -120,9 +123,13 @@ def upload_s3_folder(
                 bucket=bucket,
                 prefix=file_prefix if len(file_prefix) > 0 else None
             )
-        
 
-# TODO: Add remaining arguments from list_bucket_prefix_suffix_contains_s3
+
+# TODO: v1.0: Consider changing all instances of `local_path` to `local_prefix`.
+#  `local_path` is confusing because it could mean the full path to a local file,
+#  but it instead means the prefix to a local file. This extends to all functions in this file.
+# TODO: v1.0: Consider changing all instances of `bucket` and `prefix` to `s3_bucket` and `s3_prefix`.
+#  This will reduce ambiguity.
 def download_s3_folder(
```

Comment:
If the s3 folder contains an empty folder inside, will this function download it correctly?

Reply:
Yes because we aren't listing the folders in S3, only objects.